### ISSUE TITLE
Resolved: WhatsApp works with Google Chrome 60+

### ISSUE THREAD

**Issue by rafatosta** [rafatosta/zapzap#71]
![image](https://user-images.githubusercontent.com/18619894/180886433-3cb232a7-8e38-4686-a911-36cf87a093b6.png)
This is an old problem that happens with Whatsapp and UserAgent. I never understood the reason, but if I refresh the page it works again. I used a strategy of reloading the page and force loading from the cache, but sometimes it happens. 

### **Solution: Press F5.**

Collaborate with the project through donations by:
-  [PayPal](https://www.paypal.com/donate/?business=E7R4BVR45GRC2&no_recurring=0&item_name=ZapZap+-+Whatsapp+Desktop+for+linux%0AAn+unofficial+WhatsApp+desktop+application+written+in+Pyqt6+%2B+PyQt6-WebEngine.&currency_code=USD)
- [Ko-fi](https://ko-fi.com/X8X2E1OLG)
- [Pix](https://nubank.com.br/pagar/3c3r2/jAK5S45kuO)

**Comment by EkanshdeepGupta** [comment]
I had the same issue and was about to post a new thread, when I saw this one. What a strange issue, and an even stranger fix! Thanks for posting it here.

**Comment by bancalero** [comment]
Vim aqui para relatar exatamente esse problema hahahah Começou a acontecer comigo do nada após retornar o laptop da suspensão, até uns dias atrás não acontecia. Bom, pelo menos temos uma solução, obrigado!

**Comment by diabollich** [comment]
Downloaded google chrome, not solved i found this post, tried to use F5, nothing happens, still not solved, can somebody please help? Linux Mint Xfce 21.1

**Comment by rafatosta** [comment]
> Downloaded google chrome, not solved i found this post, tried to use F5, nothing happens, still not solved, can somebody please help? Linux Mint Xfce 21.1

Try of manually through View > Reload
![image](https://github.com/rafatosta/zapzap/assets/18619894/352c1534-8e77-4c6d-9a5a-1965405656cc)

**Comment by elppans** [comment]
Sorry to revive the post, but I thought it was better than duplicating the issue.

Today I had the exact same problem and solved it in a way that I don't know if it was coincidence.
I got this problem and F5 didn't work, not even clicking manually, as mentioned in "[F5 does not work outside the chat, #170](https://github.com/rafatosta/zapzap/issues/170)".

And I still closed it completely and opened it again and, nothing.

I was doing some research and decided to test the command via Terminal:

```
zapzap --user-agent "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.4 Chrome/113.0.5672.162 Safari/605.1.15"
```

With the command from the command line, I noticed that it opened correctly. So I closed and made the command again but without any parameter and, opened normally.
So I closed it and opened it normally through the shortcut and, it was normal.

So I thought, "What now, coinsidence?"

**Comment by rafatosta** [comment]
Yes, coincidence. The app does not accept external useragent. What you did was just run it again.

**Comment by k3ssl0sh4y** [comment]
I have the same problem, Press F5 or _View > Reload_  don´t work

**Comment by rafatosta** [comment]
> I have the same problem, Press F5 or _View > Reload_ doesn't work

Even pressing more than once?
This bug is already giving me nerves ... every time I get ready and happen again

**Comment by SGNinja** [comment]
Same problem, F5 or View>Reload doesn't work either. 
It's the first time I have the problem and it only happend on 1 of my machines running that's running debian 12

**Comment by rafatosta** [comment]
> Same problem, F5 or View>Reload doesn't work either. 
> It's the first time I have the problem and it only happend on 1 of my machines running that's running debian 12
> I try uninstalling without "--delete-data" and reinstalling it but didn't work, the only way it works for me was this:
> SOLVED:
> uninstalling with:
> flatpak uninstall --delete-data com.rtosta.zapzap
> and reinstalling with:
> flatpak install flathub com.rtosta.zapzap
> 

It happened to me yesterday and the F5 did work. I don't understand why this unpredictable behavior elsewhere. 🥲

**Comment by rafatosta** [comment]
> Same problem, F5 or View>Reload doesn't work either. 
> It's the first time I have the problem and it only happend on 1 of my machines running that's running debian 12
> I try uninstalling without "--delete-data" and reinstalling it but didn't work, the only way it works for me was this:
> SOLVED:
> uninstalling with:
> flatpak uninstall --delete-data com.rtosta.zapzap
> and reinstalling with:
> flatpak install flathub com.rtosta.zapzap
> 

It happened to me yesterday and the F5 did work. I don't understand why this unpredictable behavior elsewhere. 🥲
